tests/unit/test_charm.py

```diff
@@ -17,7 +17,31 @@
 import exceptions
 
 
-def test_sprint_boot_pebble_layer(harness: Harness, patch: SpringBootPatch) -> None:
+def help_test_application_config(expected: dict, config: dict):
+    """
+    Helper to test if config has the expected attributes
+    and that they have the correct value.
+
+    Args:
+        expected: the expected dict with the en values to test
+        config: the config to be tested
```

Comment:
s/the config to be tested/dict be tested

Reply:
Fixed